### DISCUSSION TITLE
Set Github organization to Pro for Students

### DISCUSSION THREAD

**Discussion by jacob-mannhardt** [community/community#37104] [GitHub Education]
Hey all,

I struggle to understand the Github Pro projects for students: We have a Github organization for our lab at the university and want to change our project to Pro. Even though, we can all change our personal profiles to Pro for free because we are students, we can't seem to do the same for our organization that hosts the project. Is is possible to set up a Pro organization and project for free if all developers are students and eligible for the free Pro account?

Any help is much appreciated!
Best
Jacob

**Comment by ossanna16** [comment]
Hi there @jacob-mannhardt and welcome to our community! Thank you for asking a great question 🙂

**Comment by samus-aran** [comment]
Hi @jacob-mannhardt 👋 

Sorry you are having trouble upgrading your organization.

Navigate to your [GitHub Global Campus dashboard](https://education.github.com/globalcampus/teacher). You'll find the section **Upgrade your academic organizations** near the top of the page if you have organizations you can upgrade. After clicking **Upgrade to GitHub Team**, click the Upgrade button next to the organizations you would like to upgrade.

<img width="736" alt="Animation - scroll through Global Campus. Click the GitHub link Upgrade to GitHub Team. Then click the Upgrade button next to the organizations you would like to upgrade" src="https://user-images.githubusercontent.com/10368374/168666981-4e726973-3e0f-41c2-aec0-7c9181c41830.gif">

Click the **Upgrade to GitHub Team** button and then click the **Upgrade** button next to the organization name to immediately upgrade it for free to GitHub Team.

We hope you enjoy your GitHub Education benefits and let us know how you get on 👍

**Reply by durvishbhai** [reply]
I have a organization and I own it. My education benefits are active and was approved on 10 March 2026. I can't find that option.

**Reply by Marcin-Malek** [reply]
I dont see this option either

**Comment by jvonrad** [comment]
Hello, I cant't find this anymore.

**Reply by Ronit049** [reply]
Yaaaa!!!

**Comment by GmBodhi** [comment]
I can't find it too

**Comment by jaikrishna-j** [comment]
I can't find it too

**Comment by budaizalan** [comment]
Short version: student clubs no longer get a free GitHub Team plan, and there isn’t a way to re‑enable Team‑level branch protections on a free org just because members are students. What you’re remembering did exist years ago, but **it’s no longer an available path.**

**Reply by jbhuisman** [reply]
Yeah, unfortunate but true 😔
